package.json

```diff
@@ -44,6 +44,7 @@
     "@coreui/coreui-plugin-chartjs-custom-tooltips": "^1.3.1",
     "@coreui/icons": "0.3.0",
     "@coreui/react": "^2.5.1",
+    "@types/webfontloader": "^1.6.29",
```

Comment:
types packages should go under `devDependencies` (see [Danger error](https://github.com/teamdigitale/io-onboarding-pa-fe/pull/2#issuecomment-520406194))